honeybee_ies/writer.py

```diff
@@ -2,7 +2,8 @@
 from typing import List
 
 from ladybug_geometry.geometry3d import Face3D
-from honeybee.model import Model, Shade, Room
+from honeybee.model import Model as HBModel, Shade, Room
+from dragonfly.model import Model as DFModel
```

Comment:
I agree that it seems like unnecessary work and maintenance to create a separate extension for dragonfly vs honeybee ies. So I'm ok with trying to integrate dragonfly capabilities here

However, given how you have currently implemented this, we really should change the name of the repo and package to be `dragonfly-ies` since the package is completely incapable of working without having dragonfly-core installed.

Alternatively, you can put this import statement in a try/except along with the extension of the dragonfly Model object in the `_extend_models.py` module.  You'll also need a try/except in the cli module. Then, this package can still work without needing dragonfly and dragonfly only gets extended if it is installed. So you can keep the requirements.txt with honeybee-core and maybe just include dragonfly-core in the dev-requirements if you want it available for the tests.

If you were to change the implementation to this, then I think it's fine to keep the name of the repo and package as `honeybee-ies` since it's still primarily a honeybee extension. It will just optionally extend dragonfly if it's available.

Reply:
Let me think about it. If I want to go that far and do all these checks building a separate package might be the right approach.

I'm thinking that if you can set up the repository on GitHub, I'll just move the code there and make it a separate package. It will be a small package but then we don't have to break our rules for how the dependencies are organized. That will also make the discussion about the CLI resolved by having two different entry points.

Reply:
My suggestion is, if you think that this package is eventually going to support the translation of some of the energy metadata over to IES (or any other IES-related capabilities), then it's worth it to do the separation between dragonfly and honeybee. But, if this repo is just going to be a GEM translator, then I think a new dragonfly package is overkill since you only need a couple of try/excepts to make it all work and essentially abide by our rules.